mongodbatlas/data_source_mongodbatlas_database_user.go

```diff
@@ -41,6 +41,10 @@ func dataSourceMongoDBAtlasDatabaseUser() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"ldap_auth_type": {
```

Comment:
Because it was different ticket and then I rebased it so I can use it in a test but it seems it wasn't necessary after all unless I'm using acceptance tests wrong.

Reply:
That makes sense.  Perhaps I will keep these in a single ticket to reduce confusion.   Tried to make this match how we normally do tickets in other teams but that may not make sense here.  Thank you!